src/main/java/com/microsoft/lst_bench/Driver.java

```diff
@@ -114,10 +115,21 @@ public static void main(String[] args) throws Exception {
     final TelemetryConfig telemetryConfig =
         mapper.readValue(new File(inputTelemetryConfigFile), TelemetryConfig.class);
 
+    run(taskLibrary, workload, connectionsConfig, experimentConfig, telemetryConfig);
+  }
+
+  /** Run benchmark. */
+  public static void run(
+      TaskLibrary taskLibrary,
+      Workload workload,
+      ConnectionsConfig connectionsConfig,
+      ExperimentConfig experimentConfig,
+      TelemetryConfig telemetryConfig)
+      throws Exception {
     // Create connections manager
     Map<String, ConnectionManager> idToConnectionManager = new LinkedHashMap<>();
     for (ConnectionConfig connectionConfig : connectionsConfig.getConnections()) {
-      ConnectionManager connectionManager = ConnectionManager.from(connectionConfig);
+      ConnectionManager connectionManager = InputToBench.connectionManager(connectionConfig);
```

Comment:
nit: the first time I read the class name `InputToBench`, I did not think it is dealing with config and object instantiation. The class appears to be more of a 'BenchmarkObjectFactory` to me.